### PR TITLE
Project from Trials with Github Linters

### PR DESCRIPTION
**Pull request by raminka13**
- Copy files from [Project Repo](https://github.com/raminka13/microverse-gabriel-raul)
- Add .Github/workflows/linters.yml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inspiration&family=Nanum+Gothic:wght@400;700;800&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+  <title>Albert Einstein Tribute Page</title>
+</head>
```
Comment (raminka13) [inline index.html:13]: **Webhint linter error:** _"A 'viewport' meta element was not specified."_
Please include a `<meta>` tag in the  `<head>` of the HTML, this gives the browser instructions on how to control the page's dimensions and scaling.
Example: `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        alt="" id="image">
+      <span id="img-caption" class="img-caption">Einstein in 1921, by Ferdinand Schmutzer</span>
+    </div>
+    <div class="content">
+      <p id="tribute-info" class="tribute-info">14 March 1879 - 18 April 1955) was a German-born theoretical physicist,
+        widely acknowledged
+        to be one of the greatest physicists of all time. Einstein is best known for developing the theory of
+        relativity, but he also made important contributions to the development of the theory of quantum mechanics.
+        Relativity and quantum mechanics are together the two pillars of modern physics.</p>
+      <a id="tribute-link" class="btn" href="https://en.wikipedia.org/wiki/Albert_Einstein" target="_blank">Learn More
```
Comment (raminka13) [inline index.html:56]: **Webhint linter Error:**  _Link 'rel' attribute should include 'noopener'._
Please add `rel="noopener"` inside the `<a>` tag.
`rel=”noopener”` is an HTML attribute that can be added to external links to prevent the opening page to gain any kind of access to the original page.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Setting the value of 1rem */
+html {
+    /* 1rem = 10 pixels */
+    font-size: 62.5%;
+}
```
Comment (raminka13) [inline css/style.css:18]: **Commented 🫥 code** is useful in the iterative 📱💻🖥 testing of your design, but once you are done with testing it will look unorganized 👁‍🗨. **Code that is not being used should be deleted** ⌫.
⚙︎ Please check 👀 for any commented code and delete it, that way your work will look cleaner 🧼 and more professional ✅.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+ul {
+    list-style: none;
+}
+
+img {
+    display: block;
+    width: 100%;
+    max-width: 100%;
+}
```
Comment (raminka13) [inline css/style.css:40]: **Stylelint Error:** _Expected indentation of 2 spaces_
Please check indentation throughout the file, every line of attribute should have only 2 spaces of indentation instead of 4.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    background: var(--color-primary);
+    color: var(--color-light);
+
+    position: fixed;
```
Comment (raminka13) [inline css/style.css:59]: ❌**Stylelint Error:** _Expected no more than 1 empty line_👈🏽
⚙︎ Please check 👀 for any empty lines and delete them, that way your work will look cleaner 🧼 and more professional ✅. Please remove any empty line of code 🥸.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+.btn {
+    cursor: pointer;
+    display: inline-block;
+    padding: 1rem 3rem;
+    background-color: var(--color-primary-dark);
+    color: #fff;
+    border: none;
+    border-radius: 10px;
+  }
```
Comment (raminka13) [inline css/style.css:136]: ❌**Stylelint Error:** _Expected indentation of 0 spaces_
Every whitespace ␣ is wasted size and speed ⚡️ in the deployed project. Please check any unnecessary whitespace in order to keep your code clean 🛠, lean 🔗, and efficient 📈.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+  
+  .btn:hover {
+    opacity: 0.9;
+  }
+
```
Comment (raminka13) [inline css/style.css:141]: **Stylelint Errors:** _Unexpected whitespace at end of line_ , _Unexpected missing end-of-source newline_
Please check empty lines with whitespaces.
Is expected an empty line at the end of code but in this case there is a whitespace in the line.

**Review by raminka13 — COMMENTED** [review]
**Several changes required:**
_Linters Errors and Code Clean-Up_

Please check the comments below for more detailed information.

File: css/style.css (hunk 5 above)
Comment (tresorsawasawa) [reply css/style.css:59]: - [ ] I think we can improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive and giving a reason for the requested change.
> First state the aimed problem ❗, politely offer a suggestion 😏,  conclude with a reason or importance of making that change and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

File: css/style.css (hunk 3 above)
Comment (tresorsawasawa) [reply css/style.css:18]: - [x] What do think if you improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive, politely offer a suggestion 😏, and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖. I know you can do it :muscle: :muscle:

File: css/style.css (hunk 5 above)
Comment (tresorsawasawa) [reply css/style.css:59]: - [x] I think we can improve :chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive and giving a reason for the requested change.
>  State the aimed problem ❗, politely offer a suggestion 😏,  conclude with a reason or importance of making that change, and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

File: css/style.css (hunk 6 above)
Comment (tresorsawasawa) [reply css/style.css:136]: - [x] I think it would be better if you improve chart_with_upwards_trend:   :chart_with_upwards_trend:  this review message by being more descriptive and giving a reason for the requested change.
> state the aimed problem ❗, politely offer a suggestion 😏,  conclude with a reason or importance of making that change and if possible use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Rewquired changes :recycle: :recycle: 

- [ ] I think it would be better if you add a good summary when you completed the review **by adding gifs** to make it more friendly, **highlight** some efforts improved by the student, etc. I suggest you use one template of your previous received reviews. I know you can do it :muscle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @raminka13, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
